//++++++Use of try and catch++++++++++++++
console.log("hello");
console.log("hello");
console.log("hello");
// let a = 2;
try {
    console.log(a);
} catch (e) {
    console.log(e);
}
console.log("Namaste");
console.log("Namaste");
console.log("Namaste");